tgui/packages/tgui/components/Dialog.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * @file
+ * @copyright 2022 raffclar
+ * @license MIT
+ */
+import { Box } from './Box';
+import { Button } from './Button';
+
+type DialogProps = {
+  title: string;
```

Comment:
Can effectively be `any` because restricting it to strings would prevent using fragments or html elements inside, and it's technically possible to do so.